.github/workflows/ci.yml

```diff
@@ -201,12 +211,48 @@ jobs:
         env:
           VERBOSE: false
 
+  static-checks-no-ci:
+    timeout-minutes: 30
+    name: "Static checks: no CI image"
```

Comment:
Need to change this name.

"No CI" when it's run in CI is... confusing :)

How about static-checks-simple-changes and "Static checks for simple changes only"?

Reply:
Note the "no  CI image" (it's not NO CI). I can replace it with "no image" if it is confusing. The logic is that those are all the test that we can run when there is no "CI Image" available. And we decide whether we need to build the image at all in the "selective tests" phase. I can rename "selective tests" to something else  ("Selective CI steps")? to reflect it further but I think the logic is sound:

* in the "selective" part we choose:

   a) whether we need CI image at alll
   b) whether we need to run helm tests
   c) whether we need to run unit tests
   d) whether we need to run API tests
   e) whether we need to run kubernetes tests
   f)  whether we need to run doc builds

And then all those decisions in CI should be based on those (this is currently happening but maybe the names could be adapted to reflect both indention and behaviour. 


I think if just rename "selective tests" into "selective CI steps"  "no CI image' to "no image needed", I think that would be much better and reflect the intentions WDYT?

Reply:
Actually your proposal was much better when I thought about it. Renaming it to "basic-checks"